WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -184,6 +185,41 @@ + (void)setScreenshotQuality:(NSUInteger)quality
   FBScreenshotQuality = quality;
 }
 
+// Works for Simulator and Real devices
++ (void)configureKeyboardPreference {
```

Comment:
I'd rather make the stuff configurable via appium settings

Reply:
although, this is a good idea to always set reasonable defaults (e.g. disable autocorrection and other unnecessary enhancements)

Reply:
let me add settings for non `DidShow*` items 🆗